File: odient2.m
% Define the ODE function
f = @(t, y) -y.*t + 13;

% Initial condition
y0 = 1;

% Time range
t = linspace(0, 5);

% Solve the ODE numerically using ode45
[t, y] = ode45(f, t, y0);

% Plot the solution
plot(t, y);
xlabel('Time');
ylabel('y');

File: odient2.py
import numpy as np 
from scipy.integrate import odient
import matplotlib.pyplot as plt
def returns_dydt(y,t):
	dydt=-y*t+13
	return dydt
y0=1
t=np.linspace(0,5)
y=odient(returns_dydt,y0,t)
plt.plot(t,y)
plt.xlabel("Time")
plt.ylabel("y")
plt.show()
